inject.js

```diff
@@ -50,9 +51,14 @@ return function (version, reduce, map, codec, initial) {
 
   map = map || id
   var notify = Notify()
-  return function (log, name) { //name is where this view is mounted
+  return function (log, name, opts) { //name is where this view is mounted
     var acc, since = Obv()
     var value = Obv(), state
+    var kv
+    if (opts && opts.KeyValueStore) {
+      kv = true
+      Store = SingleKeyValueStore(opts.KeyValueStore, version)
```

Comment:
Instead of flumeview-reduce knowing about key-value stores I think it would be cleaner to just have an interface for called "value store", basically it's the same as http://npm.im/atomic-file so the current `./store/fs.js`

Then we could make all the things in ./store/ into modules.

Reply:
I'm not sure I understand what you mean here. Do you mean to have an additional store type in opts that just stores one value to retrieve later? I can do that, if so.